src/components/MovieCard/MovieCard.tsx

```diff
@@ -3,7 +3,7 @@ import { Movie } from '../../types/Movie';
 import './MovieCard.scss';
 
 type Props = {
-  movie: Movie,
+  movie: Movie | null,
```

Comment:
you already handle it
![image](https://github.com/mate-academy/react_movies-list-fetch-movies/assets/91411692/a64e4191-03b2-4bd8-bf33-00b9cdada4bb)

```suggestion
  movie: Movie,
```